libs/blocks/preflight/checks/constants.js

```diff
@@ -5,6 +5,59 @@ export const STATUS = {
   EMPTY: 'empty',
 };
 
+export const CHECK_IDS = {
+
+  H1_COUNT: 'h1-count',
```

Comment:
would it make sense to expand this a little and group things within one object? e.g.

```
const CHECKS = {
  H1_COUNT: {
    id: 'h1-count'
    severity: 'warning', (or severity.warning if you want)
    seoTitle: '...',
    description: '...',
  }
}
```

that way we would have all configs grouped together, rather than tied together, this looks rather complicated: `[CHECK_IDS.H1_COUNT]: SEVERITY.WARNING,`